src/content/init.js

```diff
@@ -146,47 +146,52 @@ async function checkLoaded(settings) {
         pre.style.display = 'none';
     }
 
-    if (!loaded) {
-        flushData(settings);
-        loadedTimer = requestAnimationFrame(() =>
-            checkLoaded(settings).catch(rollbackPageChanges)
-        );
-        return;
-    }
-
     if (pre !== null) {
+        if (!loaded) {
```

Comment:
Should be placed back. In case of pre is still not loaded there will be no chance to re-check `<pre>` existence